src/core/playground/legacy/multiEd25519Account.ts

```diff
@@ -0,0 +1,30 @@
+import { HexInput } from "../../../types";
+import { AccountAddress } from "../../accountAddress";
+import type { LegacyMultiEd25519PublicKey, LegacyMultiEd25519Signature } from "./multiEd25519";
+
+export interface LegacyMultiEd25519AccountConstructorArgs {
+  publicKey: LegacyMultiEd25519PublicKey;
+  address?: AccountAddress | HexInput;
+}
+
+export class LegacyMultiEd25519Account {
```

Comment:
is there a reason that all legacy use `Account` ?

Reply:
I guess I am confused as why Legacy classes as Account, but secp256k1 doesnt.

Reply:
We never supported accounts with Secp256k1 signature schemes before

Reply:
in SDK v1 and prior to adding Secp256k1 and AnyPublicKey / AnyPrivateKey, the only available account was Ed25519 which used an authentication key derived from the Ed25519PublicKey bytes directly.

We still need to support this, and that would be LegacyEd25519Signer / Account.
For newer accounts (either Ed25519, Secp256k1 or anything that comes after) we want to use signers based on `AnyPublicKey`.

At least this is the understanding that I got chatting with people. I would not be opposed to hiding this details from the user and have all Ed25519 signers use the legacy scheme for simplicity.

Also as additional context to your question, I don't think it's even possible to have a Secp256k1 "legacy" account, I think the Rust SDK doesn't support that.

Reply:
ok thanks for explaining. I think the missing piece for me was that this PR is signers based.